cmd/core-command/res/configuration.toml

```diff
@@ -8,15 +8,14 @@ LogLevel = 'INFO'
       password = ""
 
 [Service]
-BootTimeout = 30000
-CheckInterval = '10s'
+HealthCheckInterval = '10s'
 Host = 'localhost'
-ServerBindAddr = '' # Leave blank so default to Host value unless different value is needed.
 Port = 59882
-Protocol = 'http'
-MaxResultCount = 50000
+ServerBindAddr = '' # Leave blank so default to Host value unless different value is needed.
 StartupMsg = 'This is the Core Command Microservice'
-Timeout = 45000
+MaxResultCount = 50000
+MaxRequestSize = 0 # Not curently used.
```

Comment:
what is the unit of this? KB? MB ? it should be helpful if put the unit on the comment if not on the key itself.

Reply:
Added comment to each file.